content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -261,29 +307,33 @@ Output:
 > ripple-lib@0.16.5 build /home/username/ripple-lib
 > gulp
 
-[15:22:30] Using gulpfile /home/username/ripple-lib/Gulpfile.js
-[15:22:30] Starting 'build'...
-[15:22:30] Starting 'build-debug'...
-[15:22:42] Finished 'build' after 12 s
-[15:22:42] Starting 'build-min'...
-[15:22:42] Finished 'build-debug' after 12 s
-[15:22:51] Finished 'build-min' after 9.83 s
-[15:22:51] Starting 'default'...
-[15:22:51] Finished 'default' after 4.58 μs
+[14:11:02] Using gulpfile /home/username/ripple-lib/gulpfile.js
+[14:11:02] Starting 'build'...
+[14:11:03] Starting 'build-debug'...
+[14:11:03] Starting 'build-min'...
+[14:11:18] Finished 'build-debug' after 15 s
+[14:11:18] Finished 'build' after 16 s
+[14:11:18] Finished 'build-min' after 15 s
+[14:11:18] Starting 'default'...
+[14:11:18] Finished 'default' after 19 μs
 ```
 
 This may take a while. At the end, the build process creates a new `build/` folder, which contains the files you want.
 
 The file `build/ripple-<VERSION NUMBER>.js` is a straight export of RippleAPI (whatever version you built) ready to be used in browsers. The file ending in `-min.js` is the same thing, but with the content [minified](https://en.wikipedia.org/wiki/Minification_%28programming%29) for faster loading.
 
-## Example Browser Usage
+
+
+## Demo RippleAPI in a Browser
 
 The following HTML file demonstrates basic usage of the browser version of RippleAPI to connect to a public `rippled` server and report information about that server. Instead of using Node.js's "require" syntax, the browser version creates a global variable named `ripple`, which contains the `RippleAPI` class.
 
-To use this example, you must first [build RippleAPI](#build-instructions) and then copy one of the resulting output files to the same folder as this HTML file. (You can use either the minified or full-size version.) Change the first `<script>` tag in this example to use the correct file name for the version of RippleAPI you built.
+To use this example, you must first [build a browser-compatible version of RippleAPI](#build-a-browser-compatible-version-of-rippleapi) and then copy one of the resulting output files to the same folder as this HTML file. (You can use either the minified or full-size version.) Change the second `<script>` tag in this example to use the correct file name for the version of RippleAPI you built.
 
 [**browser-demo.html:**](https://github.com/ripple/ripple-dev-portal/blob/master/content/_code-samples/rippleapi_quickstart/browser-demo.html "Source on GitHub")
 
 ```
 {% include '_code-samples/rippleapi_quickstart/browser-demo.html' %}
 ```
+
+This demo HTML loads Lodash v4.17.11 from CDNJS on Cloudflare and then loads ripple-lib v1.1.2, but you could also download and load a variant of Lodash locally. ***TODO: What version(s) of Lodash are currently supported, recommended?***
```

Comment:
I don't know. I typically use the latest version, but on the other hand, I don't use the browser version of RippleAPI very often (my test scripts are for Node.js).

Reply:
Added a doc comment to update this doc once we have required and recommended lodash versions.